indsl/equipment/operational_availability.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+
+
+def calculate_operational_availability(up_time_data: pd.Series, down_time_data: pd.Series) -> pd.Series:
+    r"""Calculate operational availability.
+
+    Calculate the operational availability of a system based on
+    the mean up time (MUT) and mean down time (MDT). The formula for the operational
+    availability is given by
+
+    .. math::
+        A_o = \frac{MUT}{MUT + MDT}
+
+    where :math:`MUT` is the mean up time and :math:`MDT` is the mean down time.
+
+
+    Parameters
```

Comment:
Please read indsl documentation on docstrings or look at any current function to check documentation style